tools/simulator/frameworks/runtime-src/proj.win32/simulator.vcxproj

```diff
@@ -179,6 +182,10 @@ xcopy /Y /Q "$(OutDir)*.exe" "$(ProjectDir)..\..\..\runtime\win32"
     </Midl>
     <PreBuildEvent>
       <Command>
+      <Command>if not exist "$(OutDir)" mkdir "$(OutDir)"
+
+xcopy /Y /Q "$(ProjectDir)..\Classes\ide-support\lang" "$(OutDir)"
+</Command>
```

Comment:
Duplicated <Command> which will cause simulator compilation failed in release mode.